Playground/SwiftShortcutsTemplate.playgroundbook/Contents/Modules/ShortcutSupport.playgroundmodule/Sources/PlaygroundPage+Shortcut.swift

```diff
@@ -0,0 +1,33 @@
+import PlaygroundSupport
+import SwiftShortcuts
+import UIKit
+
+public extension PlaygroundPage {
+    func show<S: Shortcut>(_ shortcut: S) {
+        // Construct the view controller
+        let viewController = ShortcutPreviewViewController(
+            shortcut: shortcut
+        )
+
+        // Wrap the preview within a navigation controller
+        let navigationController = UINavigationController(
+            rootViewController: viewController
+        )
+
+        // Create a new trait collection that secifies `.compact` horizontal size class
+        let traitCollection = UITraitCollection(traitsFrom: [
+            .current,
+            UITraitCollection(horizontalSizeClass: .compact)
+        ])
```

Comment:
I'm curious, what's the reasoning behind this and the trait collection override?

Reply:
Ah, so maybe I can remove this as it doesn't seem to make a difference on the iPad Version of Swift Playgrounds (but did in Xcode Playgrounds).

The idea was that because the playground preview was such a small rect, I wanted to use the iPhone style modal instead of a popover presentation and forcing a `.compact` trait collection had been a way to achieve that (I think).

But yeah, since it has no effect now I'll just remove it